### PR TITLE
feat: Add sepolia subgraph

### PR DESCRIPTION
**Pull request by ChaituVR**
Related to https://github.com/snapshot-labs/snapshot.js/issues/978

Configuration that is helpful to deploy delegation subgraph on sepolia network

### CODE REVIEW

**Comment by bonustrack** [comment]
@ChaituVR Can you add package.json script to deploy? I think all our subgraph are on hosted service? Will it work the same with Studio? Maybe we should migrate all there

**Comment by ChaituVR** [comment]
> @ChaituVR Can you add package.json script to deploy? 

Opps sorry, added now

> I think all our subgraph are on hosted service? Will it work the same with Studio? Maybe we should migrate all there

Yes, hosted service is free but not studio, should be fine for now. but we should move them in future

**Comment by tknoch-symphony** [comment]
@bonustrack what is the status with this issue? We are missing delegation subgraph for sepolia right now

File: subgraph.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-specVersion: 0.0.2
+specVersion: 0.0.4
```
Comment (ChaituVR) [inline subgraph.yaml:1]:
```suggestion
specVersion: 0.0.2
```

**Comment by ChaituVR** [comment]
Now available at https://thegraph.com/explorer/subgraphs/2SZVRR6G8txMFtf39aw2aP7BTAawRVwgqHeQXMhr7BRJ?view=Playground&chain=arbitrum-one

**Pull request merged**